tools/assetpipelineeditor/mainwindow.ui

```diff
@@ -131,7 +134,7 @@
   </widget>
   <widget class="QDockWidget" name="animationDockWidget">
    <property name="windowTitle">
-    <string>Animations</string>
+    <string>A&amp;nimations</string>
```

Comment:
is A used? would make more sense...

Reply:
it's used for "About" but maybe that one is less relevant